deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json

```diff
@@ -833,19 +810,18 @@
       {
         "allValue": null,
         "current": {},
-        "datasource": null,
-        "definition": "query_result(sum(container_memory_working_set_bytes{namespace!=\"\"}) by (namespace))",
+        "datasource": "$datasource",
+        "definition": "label_values(openebs_pool_status, namespace)",
```

Comment:
Why this change?

Reply:
This will list only the namespaces where openebs is installed because cstor pools are always formed in the openebs installed ns and using this query we can get list of only OpenEBS ns. Earlier query would give us all the ns in the cluster which we do not want.